mantidproject/mantid
# Move corrections from IDA to new UI

Fixes #12986.

To test:
- Open Indirect > Data Analysis
- Do something to see that the interface still works
  - ConvFit: sample: `irs26176_graphite002_red`, resolution: `irs26173_graphite002_res`, two Lorentzians
- Open Indirect > Corrections
- Do something to see that the interface still works
  - Apply corrections: sample: `irs26176_graphite002_red`, can: `irs26173_graphite002_red`

Release notes updates [here](http://www.mantidproject.org/index.php?title=Release_Notes_3_5_Indirect_Inelastic&diff=24350&oldid=24322).
